plugins/scaffolder-backend/src/scaffolder/tasks/logger.ts

```diff
@@ -87,9 +97,10 @@ export class WinstonLogger implements RootLoggerService {
 
     let logger = createLogger({
       level: options.level,
-      format: format.combine(redacter.format, options.format),
+      format: format.combine(options.format, redacter.format),
```

Comment:
was this part of actually fixing it, or an additional convenience?

Reply:
Actually part of fixing it, it's the main fix, the rest is just tests and cleanup to use `triple-beam` symbols to avoid any other downstream issues with other formats or transporst.